server/api.js

```diff
@@ -25,6 +25,12 @@ router.get("/desks", (req, res) => {
 	res.json(desks);
 });
 
+router.get("/users", (req, res) => {
+	db.query("SELECT * FROM desk_user", (error, result) => {
```

Comment:
maybe you could try to add an async function and use await. Very good attempt though.

Reply:
Good suggestion, I don't quite understand that method but I'll give it a go :)